src/main/java/me/tomsdevsn/hetznercloud/objects/request/LoadBalancerNetworkRequest.java

```diff
@@ -0,0 +1,22 @@
+package me.tomsdevsn.hetznercloud.objects.request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@RequiredArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class LoadBalancerNetworkRequest {
+
+    @JsonProperty("network")
+    private final long network;
```

Comment:
Not sure about the `final`, but I guess you need it because of `RequiredArgsConstructor` right?

Anyway, please use the `Long` class instead of data type `long` (capitalize `L`). Using the data type made some weird issues in the past.